server/scripts/webscraper.py

```diff
@@ -0,0 +1,125 @@
+import re
+import requests
+import urllib.request
+import time
+# from test_data import *
+from bs4 import BeautifulSoup
+
+# Started by Jeremy Lim on 20/07/2019
+
+# TO-DO LIST
+# - Get timetable done
+# - Get GenEd done
+# - Majors
+# - Degrees
+# - Add to database
+
+##### COURSES #####
+
+# To print the info of each course
+def printInfo(code, link, name, cred):
+    print("CODE: " + code) # prints the code of the course
+    print("LINK: " + link) # prints the link of the course
+    print("NAME: " + name) # prints the name of the course
+    print("CREDITS: " + cred) # prints the credits of the course
+
+# To go through each letter's links for courses
+def run_course():
+    alphabet = ['A','B','C','D','E','F','G','H','I','L','M','N','O','P','R','S','T','V','Y','Z']
```

Comment:
hard coded constants should go at top of code preferably

Reply:
You can import this: https://docs.python.org/3/library/string.html#string.ascii_uppercase

```python
from string import ascii_uppercase

```

Reply:
I think i brought this up some time ago in person with him, he is not using the whole alphabet for some reason which has eluded me now